packages/js/src/ui/components/elements/Markdown.tsx

```diff
@@ -0,0 +1,71 @@
+import { createMemo, For, JSX } from 'solid-js';
+import { useStyle } from '../../helpers';
+
+interface Token {
+  type: 'bold' | 'text';
+  content: string;
+}
+
+const parseMarkdownIntoTokens = (text: string): Token[] => {
```

Comment:
This is just parsing bold for now. In order to add italic or something more, we would have to convert to a tree.